packages/account/src/Sections/Profile/PhoneVerification/confirm-phone-number.tsx

```diff
@@ -23,19 +31,25 @@ const ConfirmPhoneNumber = observer(({ setOtpVerification }: TConfirmPhoneNumber
         setPhoneNumber(account_settings?.phone || '');
     }, [account_settings?.phone]);
 
+    React.useEffect(() => {
+        //TODOs: this will be replace to is_email_verified when we get the BE fixed
+        if (email_otp_error) {
+            setOtpVerification({ show_otp_verification: true, phone_verification_type });
+            setShouldShowPhoneNumberOTP(true);
```

Comment:
when email_otp_error why do we set "setShouldShowPhoneNumberOTP" to true?

Reply:
I'm mocking the flow for now cause we have issue with BE in the qabox it's returning error on every verification, when the issue is fixed i'll change this to use is_email_verified instead 🙏